ethereum/prices/insert_prices_from_dex_data.sql

```diff
@@ -0,0 +1,200 @@
+CREATE OR REPLACE FUNCTION prices.insert_prices_from_dex_data(start_time timestamptz, end_time timestamptz=now()) RETURNS integer
+LANGUAGE plpgsql AS $function$
+DECLARE r integer;
+BEGIN
+
+WITH trades_with_usd_amount AS (
+    SELECT
+        token_a_address as contract_address,
+        token_a_symbol as symbol,
+        decimals,
+        usd_amount/(token_a_amount_raw/10^decimals) AS price,
+        block_time
+    FROM dex.trades
```

Comment:
There seems to be a lot of missing symbols (3265) when taking the symbol from `dex.trades`. Do you think it would be more appropriate if we could use `symbol` here instead of token_a_symbol?

See this query filling the missing symbol from erc20.tokens: 

https://dune.xyz/queries/237271

I will create a PR.

Reply:
Thank you. PR merged